### DISCUSSION TITLE
1.25.1

### DISCUSSION THREAD

**Discussion by dani-garcia** [dani-garcia/vaultwarden#2621] [General]
⚠️  **Reminder**: If you are still using the `bitwardenrs/server*` Docker images, you need to migrate to the new `vaultwarden` image. Check https://github.com/dani-garcia/vaultwarden/discussions/1642 for an explanation. The old images will not receive any new updates any longer.

## What's Changed
* Updated included web vault to version 2022.6.2 by @dani-garcia 
* Sync global_domains.json by @jjlin in https://github.com/dani-garcia/vaultwarden/pull/2555
* Add TMP_FOLDER to .env.template by @fox34 in https://github.com/dani-garcia/vaultwarden/pull/2489
* Allow FireFox relay in CSP. by @BlackDex in https://github.com/dani-garcia/vaultwarden/pull/2565
* Fix hidden ciphers within organizational view. by @BlackDex in https://github.com/dani-garcia/vaultwarden/pull/2567
* Add `password_hints_allowed` config option by @jjlin in https://github.com/dani-garcia/vaultwarden/pull/2586
* Fall back to move_copy_to  if persist_to fails while saving uploaded files. by @ruifung in https://github.com/dani-garcia/vaultwarden/pull/2605
* Swap Websocket crate from ws to tungstenite, which is more maintained, supports async, and removes around 20 old duplicate versions of used crates by @dani-garcia 
* Fix armv6 issue with bullseye images by @BlackDex in https://github.com/dani-garcia/vaultwarden/pull/2491
* Add a persistent volume check. by @BlackDex in https://github.com/dani-garcia/vaultwarden/pull/2501, https://github.com/dani-garcia/vaultwarden/pull/2507
* Adding "UserEnabled" and "CreatedAt" member to the json output of a User by @Lowaiz in https://github.com/dani-garcia/vaultwarden/pull/2523
* Bump lettre to 0.10.0-rc.7 by @paolobarbolini in https://github.com/dani-garcia/vaultwarden/pull/2531
* Small email sending code improvements by @paolobarbolini in https://github.com/dani-garcia/vaultwarden/pull/2532
* A little depreciation change by @binlab in https://github.com/dani-garcia/vaultwarden/pull/2556
* Fix identicons not always working by @BlackDex in https://github.com/dani-garcia/vaultwarden/pull/2571
* Small change in log-level for better debugging by @BlackDex in https://github.com/dani-garcia/vaultwarden/pull/2577
* Address inconsistency v{version} with and without a v in the version with most recent updates.  by @nneul in https://github.com/dani-garcia/vaultwarden/pull/2595
* Bump openssl-src from 111.21.0+1.1.1p to 111.22.0+1.1.1q by @dependabot in https://github.com/dani-garcia/vaultwarden/pull/2599
* Add more clippy checks for better code/readability by @BlackDex in https://github.com/dani-garcia/vaultwarden/pull/2611
* Update deps, misc fixes and updates, small improvements on favicons and fix file-uploads by @BlackDex in https://github.com/dani-garcia/vaultwarden/pull/2543, https://github.com/dani-garcia/vaultwarden/pull/2568, https://github.com/dani-garcia/vaultwarden/pull/2619

## New Contributors
* @fox34 made their first contribution in https://github.com/dani-garcia/vaultwarden/pull/2489
* @Lowaiz made their first contribution in https://github.com/dani-garcia/vaultwarden/pull/2523
* @binlab made their first contribution in https://github.com/dani-garcia/vaultwarden/pull/2556
* @nneul made their first contribution in https://github.com/dani-garcia/vaultwarden/pull/2595
* @dependabot made their first contribution in https://github.com/dani-garcia/vaultwarden/pull/2599
* @ruifung made their first contribution in https://github.com/dani-garcia/vaultwarden/pull/2605

**Full Changelog**: https://github.com/dani-garcia/vaultwarden/compare/1.25.0...1.25.1

<hr /><em>This discussion was created from the release <a href='https://github.com/dani-garcia/vaultwarden/releases/tag/1.25.1'>1.25.1</a>.</em>

**Comment by Tieneks** [comment]
- _Updated included web vault to version **2022.6.2** by @dani-garcia_

I assume you meant **[2022.5.2](https://github.com/bitwarden/web/releases/tag/v2022.5.2)**, correct?

**Reply by dani-garcia** [reply]
The official web vault releases have been moved, they are now here: https://github.com/bitwarden/clients/releases/tag/web-v2022.6.2

**Reply by Tieneks** [reply]
Thank you. I was not aware of that.
